Update instructions (#101861)
Azure/azure-content: articles/api-management/how-to-deploy-self-hosted-gateway-kubernetes.md

5. Select the **Kubernetes** tab under **Deployment scripts**.
6. Select the **\<gateway-name\>.yml** file link and download the YAML file.
7. Select the **copy** icon at the lower-right corner of the **Deploy** text box to save the `kubectl` commands to the clipboard.

> 8. In a new terminal session run `az aks get-credentials --resource-group <resource-group-name> --name <resource-name> --admin`

Reviewer: ```suggestion
8. When using Azure Kubernetes Service (AKS), run `az aks get-credentials --resource-group <resource-group-name> --name <resource-name> --admin` in a new terminal session
```

> 8. In a new terminal session run `az aks get-credentials --resource-group <resource-group-name> --name <resource-name> --admin`

Reviewer: You might want to move 8 before 7 as the context for the next step is about using what was copied in 7

10. Paste commands to the terminal (or command) window. The first command creates a Kubernetes secret that contains the access token generated in step 4. The second command applies the configuration file downloaded in step 6 to the Kubernetes cluster and expects the file to be in the current directory.

> 11. Run the commands to create the necessary Kubernetes objects in the [default namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) and start self-hosted gateway pods from the [container image](https://aka.ms/apim/shgw/registry-portal) downloaded from the Microsoft Artifact Registry.

Reviewer: These feel like a repeat though. Maybe it's better to consolidate them?

Author: I have no preference to this one as don't have a deep K8s experience.

Reviewer: What about this?
```suggestion
10. Run the commands to create the necessary Kubernetes objects in the [default namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) and start self-hosted gateway pods from the [container image](https://aka.ms/apim/shgw/registry-portal) downloaded from the Microsoft Artifact Registry.
  - The first step creates a Kubernetes secret that contains the access token generated in step 4. Next, it creates a Kubernetes deployment for the self-hosted gateway which uses a ConfigMap with the configuration of the gateway.
```

> 8. When using Azure Kubernetes Service (AKS), run `az aks get-credentials --resource-group <resource-group-name> --name <resource-name> --admin` in a new terminal session
> 10. Run the commands to create the necessary Kubernetes objects in the [default namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) and start self-hosted gateway pods from the [container image](https://aka.ms/apim/shgw/registry-portal) downloaded from the Microsoft Artifact Registry.
>   - The first step creates a Kubernetes secret that contains the access token generated in step 4. Next, it creates a Kubernetes deployment for the self-hosted gateway which uses a ConfigMap with the configuration of the gateway.
> 12. Run the following command to check if the deployment succeeded. Note that it might take a little time for all the objects to be created and for the pods to initialize.

Reviewer: ```suggestion
8. When using Azure Kubernetes Service (AKS), run `az aks get-credentials --resource-group <resource-group-name> --name <resource-name> --admin` in a new terminal session
9. Run the commands to create the necessary Kubernetes objects in the [default namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) and start self-hosted gateway pods from the [container image](https://aka.ms/apim/shgw/registry-portal) downloaded from the Microsoft Artifact Registry.
  - The first step creates a Kubernetes secret that contains the access token generated in step 4. Next, it creates a Kubernetes deployment for the self-hosted gateway which uses a ConfigMap with the configuration of the gateway.
10. Run the following command to check if the deployment succeeded. Note that it might take a little time for all the objects to be created and for the pods to initialize.
```

    ```console
    kubectl get deployments
    ```
    It should return
    ```console
        NAME             READY   UP-TO-DATE   AVAILABLE   AGE
    <gateway-name>   1/1     1            1           18s

>     ```

Reviewer: I don't have a preference here - @dlepow will the rendering for this be OK? I don't see a preview URL so I figured asking is OK

Reviewer: I personally think it's best to keep as-is given this is how commands are typically documented.

Worst case we can prefix commands with `$ ` to show that the rest is the output of the command.

Author: I put this in there as was mildly frustrating to copy commands that included the sample output....so then had to update the command after the paste into the console.

Reviewer: It's OK.
